### DISCUSSION TITLE
How to tell if sudo.exe is installed/enabled

### DISCUSSION THREAD

**Discussion by Xilonz** [microsoft/sudo#114] [General]
It looks like sudo.exe is getting readily available across Windows devices.

Now; if I want to utilize sudo.exe in a script, how would I know its enabled in the settings app?
sudo.exe -V always exit with status 0
sudo.exe always exits with status 2
I presume I could read the stdout, but that feels hacky. 

So I'm looking so a way to check wether sudo.exe is actually usable. 

Any ideas?

**Comment by AvogatoWizardWhisker** [comment]
I wouldn't say `sudo` is available across Windows devices as it's only available in Windows 11 24h2. If you want to check if sudo is enabled, simply open Terminal and run `sudo config` and see if it's enabled or not.

**Reply by Xilonz** [reply]
`sudo config` does indeed return a proper exit code. 

Thanks for the comment 👍
